x-pack/plugins/security_solution/public/cases/containers/configure/use_connectors.tsx

```diff
@@ -31,7 +33,15 @@ export const useConnectors = (): ReturnConnectors => {
         const res = await fetchConnectors({ signal: abortCtrl.signal });
         if (!didCancel) {
           setLoading(false);
-          setConnectors(res);
+          setConnectors(
+            res.filter(
+              (connector) =>
```

Comment:
Cases get the connectors from its own API (`/api/cases/connectors/_find`) that in turn calls action's API. If you see in x-pack/plugins/case/server/routes/api/cases/configure/get_connectors.ts line 34 we filter the connectors to return only our own. I think this kind of filtering should be moved to the backend and the frontend should not deal with it.